### PR TITLE
Copy and paste a project from trials

### PR DESCRIPTION
**Pull request by telanet**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+             <li><a href="#contact">Contact</a></li>
+            
+        </ul>
+    </div>
+
+    <div id="about">
+        <h1>Hey!this is Bernad and Oliver.</h1>
+        <h3>We are software developers.</h3>
+    </div>
+    <h1 style="color:rgb(214, 12, 123)">These are some of our projects</h1>
```
Comment (telanet) [inline index.html:24]: Please move internal CSS to an external file, will make the work neat.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    <div id="projects" class="projects">
```
Comment (telanet) [inline index.html:25]: The "projects" container should be a section and not a div. Please fix that.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        
+        <div class="flex project1">
+            <h3 class="project-title">Entertainment</h3>
+            <img src="hot.png" height="300px" width="300px" alt="project1">
```
Comment (telanet) [inline index.html:29]: The image height and width should be numeric not string. Kindly check on that.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+            <a href="#"><button>View</button></a>
```
Comment (telanet) [inline index.html:30]: The button should not be a child of a(link). Please correct to make your code clean.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+             
+        </div>
+        <div class="flex project2">
+            <h3 class="project-title">Please sign up</h3>
+            <img src="copy1.png" height="300px" width="300px" alt="project1">
```
Comment (telanet) [inline index.html:35]: Please use numbers for height and width

File: index.html [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+        <div class="flex project3">
+            <h3 class="project-title">Calculator</h3>
+            <img src="sample1.png" height="300px" width="300px" alt="project1">
+            <a href="#"><button>View</button></a>
+             
+        </div>
+    </div>
+    <div class="flex project3">
+        <h3 class="project-title">Memorable Quotes</h3>
+        <img src="sample2.png" height="300px" width="300px" alt="project1">
```
Comment (telanet) [inline index.html:48]: Please use numbers for height and width.

**Review by telanet — COMMENTED** [review]
Hi @telanet

Good job so far! 👏
There are some issues that you still need to work on!

Required Changes ♻️
 The Readme should be detailed and well organized
 Check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

Check the comments under the review.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
Hi @telanet , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**